### PR TITLE
Fix path variables parsing

### PR DESCRIPTION
**Pull request by LyashenkoGS**
Before the fix the path variables parsing were broken and were failing with exceptions.

### CODE REVIEW

File: apifuzzer/swagger_template_generator.py [hunk 1]
```diff
@@ -48,9 +48,20 @@ def process_api_resources(self):
...
+                                name=param['name'],
+                                value=get_sample_data_by_type(param.get('type'))
+                            ))
+                    elif param.get('in') == ParamTypes.BODY:
+                        fuzz_type = get_fuzz_type_by_param_type(param.get('type'))
+                        template.data = fuzz_type(
+                            name=param['name'],
+                            value=get_sample_data_by_type(param.get('type'))
+                        )
+                    # TODO implement FORM_DATA, COOKIE
```
Comment (KissPeter) [inline apifuzzer/swagger_template_generator.py]: form data should be sent in body and cookie should be sent in header, isn't it?
Please do not remove else: at least put some logging there.

**Review by KissPeter — CHANGES_REQUESTED** [review]

File: apifuzzer/swagger_template_generator.py (hunk 1 above)
Comment (LyashenkoGS) [reply apifuzzer/swagger_template_generator.py]: Agree. It will be fixed.

Comment (LyashenkoGS) [reply apifuzzer/swagger_template_generator.py]: Form data and cookies parsing have been fixed. I put a logger statement in an else block but it doesn't work. I have no idea why.

**Comment by KissPeter** [comment]
Thank you, looks good

**Pull request merged**